### PR TITLE
AC-489: Non Alphanumeric Value in Postal Code Field

### PR DESCRIPTION
**Pull request by sp2710**
## Description of what I changed
Minor changes in xml file.
According to Wikipedia only alphanumeric postal code exist, so adding that line allows only alphanumeric postal codes.

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-489

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [ ] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by sp2710** [comment]
Done in next commit :)

**Comment by sp2710** [comment]
@f4ww4z apologies for that. Since i am new to this project I was not aware of that we have to claim it first.

File: openmrs-client/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -20,6 +20,7 @@
     <string name="dbname">openmrs.db</string>
     <string name="aadbname">aaopenmrs.db</string>
 
+    <string name="alphanumeric_postal_code">ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789</string>
```
[the remark this answers is not in the document]
Comment (sp2710) [reply openmrs-client/src/main/res/values/strings.xml]: I have updated the PR

**Comment by f4ww4z** [comment]
@sp2710 Name the PR like so, next time.

**Pull request merged**